kale/interpret/model_weights.py

```diff
@@ -131,3 +137,80 @@ def get_top_symmetric_weight(weights, labels, coords, p=0.001):
         sym_weights[j, i] = weight
 
     return sym_weights, top_labels, top_coords
+
+
+def select_top_features_by_masking(
```

Comment:
Is `select_top_omics_features_with_mask`? It seems specific to omics data and model